addon/components/one-way-checkbox.js

```diff
@@ -20,8 +20,9 @@ const OneWayCheckboxComponent = Component.extend(DynamicAttributeBindings, {
     'value'
   ],
 
-  click(event) {
-    invokeAction(this, 'update', this.readDOMAttr('checked'), event);
+  didInsertElement() {
```

Comment:
Not sure why this needs to change?  The top level `click` method should still be called when not using jQuery (the ember-native-dom-event-dispatcher has tests around this).